src/pages/registration/resource_type_tab/sub_type_forms/ChapterForm.tsx

```diff
@@ -69,6 +69,7 @@ const ChapterForm = () => {
             searchSubtypes={[BookType.ANTHOLOGY]}
             label={t('resource_type.chapter.published_in')}
             placeholder={t('resource_type.chapter.search_for_anthology')}
+            data-testid="search-anthology-field"
```

Comment:
sjekket at dette virker? 
tror vi må legge til egen prop for dataTestId her, som også kan brukes i `JournalForm.tsx`

Reply:
Mener dette virker, skal dobbeltsjekke

Reply:
Jeg tok feil...
Legger dataTestId på SeachContainerField